# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1420177645 18000
#      Fri Jan 02 00:47:25 2015 -0500
# Node ID 19f75d156ffee3b54e9da944c2400e9d16fbe0d1
# Parent  99522db5b911b8de88e5c4135a70451f346e53ef
don't include oct.h in Octave source files

* audiodevinfo.cc, audioread.cc: Don't include oct.h.

diff --git a/libinterp/dldfcn/audiodevinfo.cc b/libinterp/dldfcn/audiodevinfo.cc
--- a/libinterp/dldfcn/audiodevinfo.cc
+++ b/libinterp/dldfcn/audiodevinfo.cc
@@ -25,20 +25,23 @@ along with Octave; see the file COPYING.
 #endif
 
 // <cstdint> requires c++11
 #include <stdint.h>
 
 #include <string>
 #include <vector>
 
-#include "oct.h"
+#include "defun-dld.h"
+#include "error.h"
+#include "gripes.h"
+#include "oct-obj.h"
 #include "ov.h"
+#include "ov-int32.h"
 #include "ov-struct.h"
-#include "ov-int32.h"
 #include "parse.h"
 
 #if defined (HAVE_PORTAUDIO)
 #include <portaudio.h>
 #endif
 
 PaSampleFormat
 bits_to_format (int bits)
diff --git a/libinterp/dldfcn/audioread.cc b/libinterp/dldfcn/audioread.cc
--- a/libinterp/dldfcn/audioread.cc
+++ b/libinterp/dldfcn/audioread.cc
@@ -22,17 +22,21 @@ along with Octave; see the file COPYING.
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <string>
 #include <map>
 
-#include "oct.h"
+#include "defun-dld.h"
+#include "error.h"
+#include "gripes.h"
+#include "oct-obj.h"
+#include "ov.h"
 #include "ov-struct.h"
 
 #ifdef HAVE_SNDFILE
 #include <sndfile.h>
 #endif
 
 DEFUN_DLD (audioread, args, ,
   "-*- texinfo -*-\n\
